terminus-ui/form-field/src/form-field.component.html

```diff
@@ -1,4 +1,7 @@
-<div class="ts-form-field__wrapper qa-form-field">
+<div
+  class="ts-form-field__wrapper qa-form-field"
+  [ngClass]="{'ts-form-field__with-validation': !noValidationOrHint}"
```

Comment:
For BEM classes, modifier classes should use double dashes: `ts-form-field--with-validation`